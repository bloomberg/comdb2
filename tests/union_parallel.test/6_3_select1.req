select 10002 as cc union all select c from t2 union all select e from t3
