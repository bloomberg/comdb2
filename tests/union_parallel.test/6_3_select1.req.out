(cc=10002)
(cc=11)
(cc=22)
(cc=333)
